include/sys/util_macro.h

```diff
@@ -335,7 +335,35 @@ extern "C" {
 #define UTIL_AND(a, b) COND_CODE_1(UTIL_BOOL(a), (b), (0))
 
 /**
- * @brief Generates a sequence of code.
+ * @brief Generates a sequence of code with configurable separator.
+ *
+ * Example:
+ *
+ *     #define FOO(i, _) MY_PWM ## i
+ *     { LISTIFY(PWM_COUNT, (,), FOO) }
```

Comment:
Wrong order of arguments in the example: generator function should precede separator

Reply:
fixed.